tests/test_renderer_html.py

```
@@ -0,0 +1,498 @@
+        expected_files = [
+            "integration_test_story_01_intro.png",
+            "integration_test_story_02_why.png",
+            "integration_test_story_hl_01.png",
+            "integration_test_story_hl_02.png",
+            "integration_test_story_99_outro.png",
+            "integration_test_story.mp4"
+        ]
+        
+        for filename in expected_files:
+            file_path = tmp_path / filename
+            assert file_path.exists(), f"Expected file {filename} not found"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)